libs/domain/feature-loaders/src/snapshot/SnapshotProposalLoader.ts

```diff
@@ -11,8 +24,8 @@ import { withMessage } from "io-ts-types";
 
 export const URL = "https://hub.snapshot.org/graphql";
 
-export const makeQUERY: (space:String)=>DocumentNode = (space) => {
-    space = `"${space}"` // So it gets parsed correctly
+export const makeQUERY: (space: String) => DocumentNode = (space) => {
```

Comment:
If you haven't enabled the ESLint plugin for VS Code now would be a good time. It will highlight `space: String` and suggest not to use `String` (but to use `string` which is a built-in string type for Typescript). Can you fix this? (there is one later in this file too).